support/scripts/mkgrubimg

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+# help menu
+usage()
+{
+	echo "Usage: $0 [options] [output file]"
+	echo "Creates bootable GRUB images from Unikraft kernel images."
+	echo ""
+	echo "  -h                         Display help and exit"
+	echo "  -f                         Output format (supported: iso)"
+	echo "  -k                         Path to the Unikraft kernel image"
+	echo "  -a                         Kernel command-line parameters (optional)"
+	echo "  -i                         Path to initrd cpio file (optional)"
+	exit 1
+}
+
+# default options
+OPTFORMAT="iso"
+OPTCMDLINE=""
+
+# cleanup
+BUILDDIR=
+sighandler_exit()
+{
+	if [ ! -z "${BUILDDIR}" -a -d "${BUILDDIR}" ]; then
+		rm -rf "${BUILDDIR}"
+	fi
+}
+
+# process options
+while getopts "hk:i:f:a:" OPT; do
+	case "${OPT}" in
+		h)
+			usage
+			;;
+		f)
+			OPTFORMAT="${OPTARG}"
+			;;
+		k)
+			OPTKERNELIMG="${OPTARG}"
+			;;
+		a)
+			OPTCMDLINE="${OPTARG}"
+			;;
+		i)
+			OPTINITRD="${OPTARG}"
+			;;
+		*)
+			usage
+			;;
+	esac
+done
+shift $((OPTIND-1))
+
+# validate arguments
+if [[ -z "${OPTKERNELIMG}" || -z "$1" ]]; then
+	usage
+fi
+
+# Register exit handler and create BUILDDIR
+trap sighandler_exit exit
+BUILDDIR="$( mktemp -d )"
+if [ $? -ne 0 -o -z "${BUILDDIR}" -o ! -d "${BUILDDIR}" ]; then
+	echo "Failed to create temporary directory" 1>&2
+	exit 1
+fi
+
+OPTKERNELNAME="${OPTKERNELIMG##*/}"
+OPTINITRDNAME="${OPTINITRD##*/}"
+
+case "${OPTFORMAT}" in
+	# generate grub iso image
+	iso)
+		# configure grub
+		mkdir -p "${BUILDDIR}/boot/grub"
+
+		if [ ! -z "${OPTINITRD}" ]; then
+			GRUB_CFG_INITRD="module /boot/${OPTINITRDNAME}"
+			cp "${OPTINITRD}" "${BUILDDIR}/boot/${OPTINITRDNAME}"
```

Comment:
Sorry, I did not see this before: We should probably check return code of the commands and return from the script with an error as well. Otherwise we have no way to detect if the iso was successfully created if we include the script in a Makefile, for example.
```bash
cp "${OPTINITRD}" "${BUILDDIR}/boot/${OPTINITRDNAME}" || exit 1
```